### ISSUE TITLE
npm i -g raml2wiki  doesn't work on osx

### ISSUE THREAD

**Issue by danielb2** [jhitchcock/raml2wiki#2]
Need to open the executable and then save it as a unix file in order to execute. Should be fixed.

``` bash
[~/t/raml2wiki]$ npm i -g raml2wiki
/Users/daniel/.nvm/current/bin/raml2wiki -> /Users/daniel/.nvm/current/lib/node_modules/raml2wiki/lib/raml2wiki.js
raml2wiki@0.1.2 /Users/daniel/.nvm/current/lib/node_modules/raml2wiki
├── commander@2.6.0
├── handlebars@2.0.0 (optimist@0.3.7, uglify-js@2.3.6)
└── raml2obj@0.3.0 (raml-parser@0.8.10)
[daniel@nero]--(09:21:59)
[~/t/raml2wiki]$ raml2wiki
env: node\r: No such file or directory
```

**Comment by jhitchcock** [comment]
Thnks for the issue. I'll look into it ASAP and update here

**Comment by jraede** [comment]
@jhitchcock I have this problem as well. You are using DOS line endings and they are unusable by *nix systems. A temporary solution is to `brew install dos2unix` and then `dos2unix node_modules/.bin/raml2wiki` but it's pretty hacky.

**Comment by sigmundlundgren** [comment]
Need the fix as well, any solution yet?

**Comment by PLPeeters** [comment]
I can confirm what @jraede said above. The issue is the usage of DOS line endings. Everything works fine after converting line endings to *nix ones.

**Comment by jhitchcock** [comment]
Ah sorry for the delay, I have been away. I'll fix that this weekend :)

**Comment by jefferai** [comment]
Just ran into this. Still broken.

**Comment by jhitchcock** [comment]
Fixed in 0.1.3 just published

jhk_1@b:~/workspace $ npm i -g raml2wiki


/home/ubuntu/.nvm/v0.10.35/bin/raml2wiki -> /home/ubuntu/.nvm/v0.10.35/lib/node_modules/raml2wiki/lib/raml2wiki.js
raml2wiki@0.1.3 /home/ubuntu/.nvm/v0.10.35/lib/node_modules/raml2wiki
├── commander@2.7.0 (graceful-readlink@1.0.1)
├── handlebars@2.0.0 (optimist@0.3.7, uglify-js@2.3.6)
└── raml2obj@0.3.0 (raml-parser@0.8.10)


jhk_1@b:~/workspace $ raml2wiki

Error: You need to specify the RAML input file

  Usage: raml2wiki [options] [RAML input file]
